vendor/monkey-patches.js

```diff
@@ -21,7 +21,7 @@
       // that happen _between_ acceptance tests will always share
       // `pendingRequests`.
       _Ember.Application.reopen({
-        willDestroy() {
+        willDestroy: function() {
```

Comment:
with https://github.com/emberjs/ember-test-helpers/pull/271 this should not be necessary anymore